### DISCUSSION TITLE
How to find to get  free ethereum for Rinkeby Test Network

### DISCUSSION THREAD

**Discussion by nizomjon9909** [smartcontractkit/full-blockchain-solidity-course-py#433] [Q&A] [answered]
hey  guys how to get  free ethereum for Rinkeby Test Network , I have tried a couple of times  so far no luck , can skip this part

**Comment by cromewar** [comment, marked as answer]
Hello @nizomjon9909 
Use chainlink faucet: https://faucets.chain.link/rinkeby
Or copy paste your address here I'll sent you some ether to Rinkeby.

**Reply by nizomjon9909** [reply]
sorry I had  only an  Ethereum mainnet   in metamask    ...  do you know  how to  add Rinkeby Test Network  ? 
after that I will send you my address

**Reply by cromewar** [reply]
Just click here on Mainnet an change to rinkeby, anyways the address will be the same.

![image](https://user-images.githubusercontent.com/20199180/143298320-1f6b5b43-51d8-48c6-8d15-02093d9dbf62.png)

**Reply by nizomjon9909** [reply]
0x47300cbc0831bf6A78319d6c8b390aD18E1650f3     there is no any option to change the  rinkeby   ,  please send  if its possible

**Reply by cromewar** [reply]
Just click the arrow and you'll see this menu with all the networks

![image](https://user-images.githubusercontent.com/20199180/143298858-f6ede5db-3a5a-4c4d-a5ee-d461d799a232.png)

I sent you 1 Eth, here is the transaction hash: 0xc10e3086dc70bd558ed041d47d1ca99b2da23f4eb64f64e37277c286adb52f60

I'll close this thread now, best regards, Cromewar.

**Comment by nizomjon9909** [comment]
![2021-11-24 20 43 34](https://user-images.githubusercontent.com/77353214/143303601-24c4cf10-07ff-4365-9141-c68ce16aa1ca.jpg) 
 thank you so  much , I   will try to figure out  that :

**Reply by cromewar** [reply]
That's weird as Metamask already comes with rinkeby,  I suggest to reinstall the extension ;)

**Reply by programonauta** [reply]
> thank you so much , I will try to figure out that :

Hi if you click on Show/Hide Testnet networks the Rinkeby (and others testnet networks) will appear.

![image](https://user-images.githubusercontent.com/6033539/145623747-baa25157-3f70-45e6-9163-e317898f0b8c.png)

**Reply by cromewar** [reply]
Nice Hint @programonauta thanks.
